rqt_gui/package.xml

```diff
@@ -12,9 +12,18 @@
 
   <author>Dirk Thomas</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend version_gte="0.3.0">qt_gui</build_depend>
-  <run_depend>catkin</run_depend>
-  <run_depend version_gte="0.3.0">qt_gui</run_depend>
+
+  <exec_depend>ament_cmake_python</exec_depend>
```

Comment:
Why is this an `exec_depend`?

Reply:
Looks like it was supposed to be ament_index_python

Reply:
Addressed by #153.